src/appengine/libs/request_cache.py

```diff
@@ -15,12 +15,21 @@
 
 from builtins import str
 import collections
+import flask
 
 from base import memoize
 from libs import auth
 from metrics import logs
 
 
+def get_cache_backing():
+  """Get the cache backing for saving current context data."""
+  # TODO(singharshdeep): Remove get_current_request() after flask migration.
+  if flask.request:
+    return flask.g
+  return auth.get_current_request()
```

Comment:
nit: move get_current_request here too. auth.get_current_request wasn't intended to be used from elsewhere and doesn't make sense as a place for this.